examples/cmake/FindGeodeNative.cmake

```diff
@@ -31,15 +31,14 @@
 #
 # This module will set the following variables in your project:
 #
-# ``GEODE_NATIVE_INCLUDE_DIRS``
+# ``GEODE_NATIVE_CPP_INCLUDE_DIRS``
 #   where to find CacheFactory.hpp, etc.
 # ``GEODE_NATIVE_LIBRARIES``
 #   the libraries to link against to use Geode Native.
 # ``GEODE_NATIVE_FOUND``
 #   true if the Geode Native headers and libraries were found.
 #
```

Comment:
Need to verify there names are still valid and update the docs to include the dotnet component.